src/styles/index.css

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif, Arial;
```

Comment:
The font-family property should list generic font families at the end. Consider placing 'Arial' before 'sans-serif'.